packages/connectivity/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -5,11 +5,18 @@
 
     <application android:name="io.flutter.app.FlutterApplication" android:label="connectivity_example" android:icon="@mipmap/ic_launcher">
         <activity android:name=".MainActivity"
-                  android:launchMode="singleTop"
-                  android:theme="@android:style/Theme.Black.NoTitleBar"
-                  android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection"
-                  android:hardwareAccelerated="true"
-                  android:windowSoftInputMode="adjustResize">
+            android:launchMode="singleTop"
+            android:theme="@android:style/Theme.Black.NoTitleBar"
+            android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection"
+            android:hardwareAccelerated="true"
+            android:windowSoftInputMode="adjustResize">
+        </activity>
+        <activity android:name="dev.flutter.plugins.connectivity_exmaple.MainActivity"
```

Comment:
FYI, in Java/Android, packages usually don't include underscores. You might choose `plugins.connectivity.example` or `plugins.connectivityexample`.

Reply:
using connectivityexample.